InvenTree/common/forms.py

```diff
@@ -145,12 +145,14 @@ def clean(number):
                 for col in row['data']:
                     # Get column matching
                     col_guess = col['column'].get('guess', None)
+                    # Set field name
+                    field_name = col_guess.lower() + '-' + str(row['index'])
 
                     # Create input for required headers
                     if col_guess in file_manager.REQUIRED_HEADERS:
-                        # Set field name
-                        field_name = col_guess.lower() + '-' + str(row['index'])
                         # Set field input box
+
+                        # TODO maybe not here but in an own function?
```

Comment:
Could be a good idea to break it down by the type of file ('order', 'part', etc)